components/Modal/modal.css

```diff
@@ -0,0 +1,30 @@
+@import "/docs.css";
+@import "../Utilities/utilities.css";
+@import "../Button/button.css";
+
+.modal{
+    position: absolute;
+    top:20%;
+    left:50%;
+    margin-left:-15vw;
+    width:30vw;
+    max-width: 100%;
+    border: 2px solid #808080;
+    padding: 1rem;
+    background-color: var(--white-color);
+}
+
+.modal-header{
+    display: flex;
+    justify-content: space-between;
+}
+
+.modal-footer{
+    display: flex;
+    justify-content: end;
+}
+
+.close-icon{
+    font-size: 1.5rem;
+    color: grey;
```

Comment:
Try using Hex color extension.. since its text color variable would be a great choice